background/sync-manager.js

```diff
@@ -208,6 +209,7 @@ const syncMan = (() => {
       setState(drive, state) {
         return chromeLocal.setValue(STORAGE_KEY + drive.name, state);
       },
+      retryMaxAttempts: 0,
```

Comment:
@tophf Since there is already a retry mechanic in Stylus, I disabled this feature in db-to-cloud. Should use it?

Reply:
Now that db-in-cloud can retry, we should probably rely on it, and remove this code from Stylus.